app/views/users/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<!-- <form accept-charset="UTF-8" method=user_path action="/users">
```

Comment:
###### commented out `HTML form`

* [ ] uses `users_path` helper for action 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

- what I wanted here is for you to change line 1 from:
`<!-- <form accept-charset="UTF-8" method=user_path action="/users">`
to:
`<!-- <form accept-charset="UTF-8" method=user_path action=users_path>`

Reply:
Thanks for your suggestions. changes made